### ISSUE TITLE
Merge the 'Authorized Support Ticket Initiators' airtable into the 'Community Partners' airtable

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/infrastructure#3698]
Split out of https://github.com/2i2c-org/infrastructure/issues/3048 to handle this specific task.

This task can be marked as completed when all the information in https://airtable.com/appxk7c9WUsDjSi0Q/tbl3CWOgyoEtuGuIw/viwtpo7RxkYv63hiD?blocks=hide (which was gathered in the process described in https://github.com/2i2c-org/infrastructure/issues/3048#issuecomment-1907195217) is merged into https://airtable.com/appbjBTRIbgRiElkr/tblYGygEo5PQBSUYt/viwGZLBERreDcvUMO?blocks=hide in a useful form.

Once that is merged, a follow-up to https://github.com/2i2c-org/team-compass/pull/797 and https://github.com/2i2c-org/infrastructure/pull/3634 needs to be made to point to the new location.

**Comment by yuvipanda** [comment]
Assigning to @jmunroe per https://github.com/2i2c-org/infrastructure/issues/3048#issuecomment-1909011115

**Comment by jmunroe** [comment]
I've merged all of the data from the "Authorized Support Ticket Initiators" base into the "2i2c" base.  The data itself is stored in the Communities table with a list of "Authorized Technical Contacts" which has a many-to-many relationship with the 'Contacts' table to store email addresses.  To update this data (currently) requires adding/removing/modifying records in the 2i2c base.  

I think it makes sense for Partnerships to take responsibility for keeping the data consistent with the leads, contracts, and relationships we have with communities.  

For use by Support Triagers, I suggest the Airtable interface: [Airtable: Support Use](https://airtable.com/appbjBTRIbgRiElkr/pagog9egWRpeLyLGW?uMtyG=b%3AWzAsWyI4WnV1VyIsOSxbInNlbDRjSG1pdlRuRlRza0ljIl0sIklPZVJqIl1d ) be the primary way of looking up this information when triaging FreshDesk Support Tickets.  I'll create a short video overview of what that interface currently looks like.

In a future state, I can imagine an integration between FreshDesk and Airtable to make this same information appear within FreshDesk but that is beyond the scope of this particular issue.

**Comment by jmunroe** [comment]
Video overview of work done: https://www.loom.com/share/5631fdd559f84363a9873f54cc4e88fa?sid=d84120e1-9f39-4e56-9f61-d3a08d5f8188
